4_advanced_usage/40_node_properties.txt

```diff
@@ -0,0 +1,53 @@
+=== Node properties
+
+==== Defining properties
+
+Node properties can be found in the "properties" tab of each node in Rudder.
+
+Node properties can be modified using Rudder API, see http://www.rudder-project.org/rudder-api-doc/#api-Nodes-updateNodeProperties
+
+Currently a property is interpreted as a string value. So properties can be seen as a list of key=value pairs.
+
+==== Using properties
+
+You can use a node property almost everywhere in Rudder:
+- in directive parameters
+- in the technique editor
+- in your own CFEngine code
+
+To use a property, simply use the variable node.properties with the variable call syntax.
+
+Example with a string property named mykey:
+
+----
+
+${node.properties[mykey]}
```

Comment:
Please use real-life examples wherever possible. "datacenter=Paris" is a good example (here, just replace "mykey" by "datacenter" twice)